src/quicer.erl

```diff
@@ -951,6 +953,14 @@ do_forward_stream_msgs(Stream, Owner, MRef) ->
             ok
     end.
 
+%% @doc garbage collect some quic event that is useless to the caller.
+-spec flush(atom(), handle()) -> ok.
+flush(QuicEventName, Handle) when is_atom(QuicEventName) ->
+  receive
+    {quic, QuicEventName, Handle, _} -> ok
```

Comment:
if it should have been delivered already,
maybe raise an error exception instead of blocking forever.

Reply:
The event is sent from NIF code, not from another process.
It is like an assertion too, the event must be in the mailbox or comes late from NIF.

yes maybe have some reasonable timeout and raising exceptions is much nicer than blocking forever.